examples/do-not-use/docs/karpenter.md

```diff
@@ -0,0 +1,99 @@
+# Karpenter
```

Comment:
This will eventually live in the docs folder of `terraform-aws-eks-blueprints-addons` when the changes are sync'd back to that repo